src/main/java/htsjdk/tribble/index/AbstractIndex.java

```diff
@@ -343,13 +343,20 @@ public void write(final LittleEndianOutputStream stream) throws IOException {
     }
 
     @Override
-    public void writeBasedOnFeatureFile(final File featureFile) throws IOException {
-        if (!featureFile.isFile()) return;
+    public void write(final File idxFile) throws IOException {
         final LittleEndianOutputStream idxStream =
-                new LittleEndianOutputStream(new BufferedOutputStream(new FileOutputStream(Tribble.indexFile(featureFile))));
-        write(idxStream);
-        idxStream.close();
+                new LittleEndianOutputStream(new BufferedOutputStream(new FileOutputStream(idxFile)));
```

Comment:
It comes from the previous version, but I've just put everything in a single line.